executor/executor_pkg_test.go

```diff
@@ -50,84 +45,6 @@ var (
 	InspectionRules        = inspectionRules
 )
 
-func TestShowProcessList(t *testing.T) {
```

Comment:
Why is it removed?

Reply:
1. executor depends on testkit, which will cause circular reference
2. test for `show processlist` is covered by other tests, and this `old` test isn't a good test.